osu.Game/Beatmaps/IWorkingBeatmap.cs

```diff
@@ -79,5 +115,20 @@ public interface IWorkingBeatmap
         /// </summary>
         /// <param name="storagePath">The storage path to the file.</param>
         Stream GetStream(string storagePath);
+
+        /// <summary>
+        /// Beings loading the contents of this <see cref="IWorkingBeatmap"/> asynchronously.
+        /// </summary>
+        public void BeginAsyncLoad();
```

Comment:
I dunno about this stuff... The whole "load" stuff is very game-specific.

My hope is to eventually revert these changes after you're done with things, so let's go with it as it is.

Reply:
What's the expectation of `IWorkingBeatmap`? No async handling at all? I can get behind that, but needs a lot of refactoring work, at a much later stage.